benchmarks/startup/scenario.py

```diff
@@ -0,0 +1,32 @@
+import subprocess
+
+import bm
+
+
+class Startup(bm.Scenario):
+    telemetry_enabled = bm.var_bool()
+    use_ddtrace_run = bm.var_bool()
+    with_trace = bm.var_bool()
+
+    def run(self):
+        cmd = "python"
+        if self.use_ddtrace_run:
+            cmd = "ddtrace-run python"
+
+        code = "from ddtrace import tracer;"
+        if self.with_trace:
+            code += "with tracer.trace('test'): pass;"
+
+        def _(loops):
+            for _ in range(loops):
+                subp = subprocess.Popen(
```

Comment:
you can probably just use `subprocess.check_output([...])` or `subprocess.check_call([...])`